### PR TITLE
Remove old PNI fake data, and add Percy products back in

### LINKED ISSUE #5876: PNI follow-up: create new Percy products
- Set up percy-specific testing data for the new wagtail based PNI
  - [ ] percy-specific general product
  - [ ] percy-specific software product

### PR DESCRIPTION
**Pull request by Pomax**
Closes #5876 by adding general and software Percy products, with the Cypress tests turned back on, too.

Also comments off old PNI product generation.

### CODE REVIEW

File: network-api/networkapi/buyersguide/factory.py [hunk 1]
```diff
@@ -364,39 +361,106 @@ class Meta:
...
+        title='General Percy Product',
+        first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        parent=pni_homepage,
+        # product fields
+        privacy_ding=True,
+        adult_content=True,
+        uses_wifi=True,
+        uses_bluetooth=True,
+        review_date=datetime(2025, 1, 1, tzinfo=timezone.utc),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/factory.py:377]: So, this doesn't actually seem to set the right date. @KalobTaulien any idea what the correct syntax would be?

Comment (KalobTaulien) [reply network-api/networkapi/buyersguide/factory.py:377]: Ahh, review_date is using a DateField, whereas the others are using a DateTimeField().

```python
import datetime 
```

```suggestion
        review_date=datetime.date(2025, 1, 1),
```

File: network-api/networkapi/buyersguide/factory.py [hunk 2]
```diff
@@ -412,43 +476,17 @@ def generate(seed):
...
-    )
-
     reseed(seed)
 
     print('Generating Buyer\'s Guide product updates')
     generate_fake_data(ProductUpdateFactory, 15)
 
+<<<<<<< HEAD
+    # TODO: link updates into products
+
```
Comment (KalobTaulien) [inline network-api/networkapi/buyersguide/factory.py:486]: Git conflict

**Review by KalobTaulien — CHANGES_REQUESTED** [review]

File: network-api/networkapi/buyersguide/factory.py [hunk 3]
```diff
@@ -364,39 +361,106 @@ class Meta:
...
+        title='Software Percy Product',
+        first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        parent=pni_homepage,
+        # product fields
+        privacy_ding=True,
+        adult_content=True,
+        uses_wifi=True,
+        uses_bluetooth=True,
+        review_date=datetime(2025, 1, 1, tzinfo=timezone.utc),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/factory.py:424]:
```suggestion
        review_date=date(2025, 1, 1),
```

File: network-api/networkapi/buyersguide/factory.py [hunk 4]
```diff
@@ -412,43 +476,17 @@ def generate(seed):
...
 
     print('Generating Buyer\'s Guide product updates')
     generate_fake_data(ProductUpdateFactory, 15)
 
+<<<<<<< HEAD
+    # TODO: link updates into products
+
+=======
+>>>>>>> e820535e38da1d7831877f2d56d166ba1418cb5e
+    """
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/factory.py:489]:
```suggestion
    # TODO: link updates into products

```

File: network-api/networkapi/buyersguide/factory.py [hunk 5]
```diff
@@ -364,39 +361,106 @@ class Meta:
...
+        title='General Percy Product',
+        first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        parent=pni_homepage,
+        # product fields
+        privacy_ding=True,
+        adult_content=True,
+        uses_wifi=True,
+        uses_bluetooth=True,
+        review_date=date(year=2025, month=1, day=1),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/factory.py:377]: this still doesn't work correctly.

File: network-api/networkapi/buyersguide/factory.py [hunk 6]
```diff
@@ -364,39 +361,106 @@ class Meta:
...
+        title='Software Percy Product',
+        first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+        parent=pni_homepage,
+        # product fields
+        privacy_ding=True,
+        adult_content=True,
+        uses_wifi=True,
+        uses_bluetooth=True,
+        review_date=date(year=2025, month=1, day=1),
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/factory.py:424]: this still doesn't work correctly.

**Review by KalobTaulien — APPROVED** [review]

**Comment by Pomax** [comment]
removed a bunch of dead tests due to removing the old PNI's fake data.

**Review by KalobTaulien — APPROVED** [review]

**Pull request merged**